_topics/en/cve.md

```diff
@@ -0,0 +1,186 @@
+
+  - title: "CVE-2017-12842: vulnerability in Bitcoin SPV proofs"
+    url: /en/newsletters/2018/12/28/#cve-2017-12842
+
+  - title: "CVE-2021-31876: discrepancy between BIP125 and Bitcoin Core implementation"
+    url: /en/newsletters/2021/05/12/#cve-2021-31876-discrepancy-between-bip125-and-bitcoin-core-implementation
+
+## Optional.  Same format as "primary_sources" above
+see_also:
+  - title: CVE-2018-17144 duplicate inputs vulnerability
+    link: topic cve-2018-17144
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Common Vulnerabilities and Exposures (CVEs)** are serious
+  vulnerabilities that have been cataloged to help developers,
+  researchers, and the public efficiently share information about
+  potential threats.
+
+extra:
+  cves:
+    CVE-2020-26896:
+      link: /en/newsletters/2020/10/28/#cve-2020-26896-improper-preimage-revelation
+      summary: >
+        Vulnerability in LND due to improperly revealing local preimages
+        in response to foreign routing requests, which could've allowed
+        funds to be stolen.  Fixed in LND 0.11.0-beta.
+
+    CVE-2020-26895:
+      link: /en/newsletters/2020/10/28/#cve-2020-26895-acceptance-of-non-standard-signatures
+      summary: >
+        Vulnerability in LND due to accepting non-standard "high-S"
+        signatures, which could've prevented channels from closing in a
+        timely manner and so allowed funds to be stolen.  Fixed in LND
+        0.10.0-beta.
+
+    CVE-2018-17145:
+      link: /en/newsletters/2020/09/16/#inventory-out-of-memory-denial-of-service-attack-invdos
+      summary: >
+        Flooding a node with an excess of `inv` messages could lead to
+        the node running out of memory and crashing, increasing the risk
+        of [eclipse attacks][topic eclipse attacks] that can steal
+        money.  Affected Bitcoin Core, Btcd, Bcoin, and multiple
+        altcoin nodes.  Fixed in Bitcoin Core 0.16.2, Btcd 0.21.0-beta,
+        and Bcoin 1.0.2.
+
+    CVE-2017-12842:
+      link: /en/newsletters/2018/12/28/#cve-2017-12842
+      summary: >
+        Allows creating an SPV proof for a transaction that doesn’t
+        exist by specially crafting a real 64-byte transaction that gets
+        confirmed in a block.  Can be used to steal from SPV clients,
+        although other known attacks against SPV clients are cheaper.
+        Proposed to be fixed in the [consensus cleanup][topic consensus
+        cleanup] soft fork.
+
+    CVE-2017-18350:
+      link: /en/newsletters/2019/11/13/#cve-2017-18350-socks-proxy-vulnerability
+      summary: >
+        A vulnerability in Bitcoin Core affecting users of SOCKS
+        proxies.  Fixed in Bitcoin Core 0.16.0.
+
+    CVE-2013-2292:
+      link: https://bitcointalk.org/?topic=140078
+      summary: >
+        A vulnerability in Bitcoin Core that it was believed could've
```

Comment:
I'm normally fine with contractions, but for some reason "could've" seems too informal. I know that's inconsistent and probably just a personal thing, so feel free to ignore.

Reply:
Nah, my spell checker doesn't think they're real contractions either; they're just such a part of my causal writing and speech patterns that I forget they're not normal for the seven billion people who aren't Piney hicks.  I'll expand the term.